doc/dev/release-checklist.rst

```diff
@@ -132,7 +132,49 @@ Getting Started
    for screed are to run the khmer automated tests with the new version of
    screed installed and then to run the khmer acceptance tests.
 
-#. Make sure any release notes are merged into doc/release-notes/.
+#. Make sure any release notes are merged into doc/release-notes/. Release
+   notes should be written in the `.md` format to allow for GitHub rich text
```

Comment:
.md is only because of the requirements for the GitHub release notes. GitHub will happily render and edit many types of plain text markup like Markdown and reStructuredText.